memdown.js

```diff
@@ -225,4 +225,4 @@ MemDOWN.destroy = function (name, callback) {
   setImmediate(callback)
 }
 
-module.exports = MemDOWN
+module.exports = MemDOWN.default = MemDOWN
```

Comment:
To support `import { MemDown } from 'memdown'`, we need to add another propertie like this:
```diff
- module.exports = MemDOWN.default = MemDOWN
+ module.exports = MemDOWN.default = MemDOWN.MemDOWN = MemDOWN
```

Reply:
its already supported (see line 3 export of MemDown). 

```
import {MemDown, MemDownGetOptions, MemDownIteratorOptions} from './memdown'; 
```

Reply:
Yes, got it.